config/custom-environment-variables.json

```diff
@@ -7,5 +7,8 @@
     },
     "api_key" : {
         "SENDGRID": "SENDGRID_API_KEY"
+    },
+    "email_template" : {
```

Comment:
I think you've stored this in Heroku Config. However, this template-id is generally not sensitive data. so maybe putting this in `default.json` would be simpler. 

EDIT: No, it's actually good idea here in Heroku Config. So we can change the template-id without deploying new codebase in case we want to use a different template. All good!